cht/Atari - Lynx/Ninja Gaiden (1990).cht

```diff
@@ -0,0 +1,21 @@
+cheat0_address = "20244"
+cheat0_address_bit_position = "255"
+cheat0_big_endian = "false"
+cheat0_cheat_type = "1"
+cheat0_code = ""
+cheat0_desc = "my cheat"
```

Comment:
Your cheat?

Reply:
Good catch. Could have sworn I had fixed that. Will do!!